ext_mongodb.php

```diff
@@ -21,6 +21,11 @@ public function getMessage() : string
 		return $this->message;
 	}
 
+	public function getInfo() : array
```

Comment:
We simply have a proto that reads `array|null`. AFAIK, we're not using return type hints for PHP 7.